test/system/admin/users_test.rb

```diff
@@ -0,0 +1,28 @@
+require "admin_system_test_case"
+
+class Admin::UsersTest < AdminSystemTestCase
+  def setup
+    login_as users(:super_admin)
+  end
+
+  test "listing users" do
+    visit admin_url("users")
+
+    assert_selector "h1", text: /#{User.model_name.human(count: User.count)}/i
```

Comment:
Somewhat convoluted matching due to the lowercasing of the h1 via CSS and I don't want tests to break if we stop doing that. Probably worth adding a `assert_selector_case_insensitive` method in the future once we have more usages.